src/footerbranding.component.tsx

```diff
@@ -1,76 +1,143 @@
-            <FontAwesomeIcon icon="chevron-up" size="2x" />
-          </a>
-        </div>
-
-        <div className="col-lg-4">
-          <div className="summary row">
-            <div className="col">
-              <h2>
-                <a href="#">Serlo.org</a> ist die Wikipedia fürs Lernen.
-              </h2>
-              <p>
-                Wir sind eine engagierte Gemeinschaft, die daran arbeitet,
-                hochwertige Bildung weltweit frei verfügbar zu machen.
-              </p>
-              <p>
+      <BrandingGrid fluid>
+        <BrandingWrap>
+          <Col xs={12} md={8} className="brand">
+            <Logo serloSlogan={this.props.serloSlogan}/>
+            <a id="toplink" href="#top" title="Mit Serlo ganz nach oben ;)">
+              <FontAwesomeIcon icon="chevron-up" size="2x" />
+            </a>
+          </Col>
+
+          <Col xs={12} md={4}>
+            <Row className="summary">
+              <Col xs={12}>
+                <h2>
+                  <a href="#">Serlo.org</a> ist die Wikipedia fürs Lernen.
+                </h2>
+                <p>
+                  Wir sind eine engagierte Gemeinschaft, die daran arbeitet,
+                  hochwertige Bildung weltweit frei verfügbar zu machen.
+                </p>
+                <p>
+                  <a href="#" className="icon">
+                    <FontAwesomeIcon icon="chevron-circle-right" />{' '}
+                    <span>Mehr erfahren</span>
+                  </a>
+                </p>
+              </Col>
+            </Row>
+
+            <Row className="support">
+              <Col xs>
                 <a href="#" className="icon">
-                  <FontAwesomeIcon icon="chevron-circle-right" />{' '}
-                  <span>Mehr erfahren</span>
+                  <img alt="part" src={participateSrc} />
+                  <span>Mitmachen</span>
                 </a>
-              </p>
-            </div>
-          </div>
-
-          <div className="support row">
-            <div className="col">
-              <a href="#" className="icon">
-                <img alt="part" src={participateSrc} />
-                <span>Mitmachen</span>
-              </a>
-            </div>
-            <div className="col">
-              <a href="#" className="icon">
-                <img alt="part" src={donateSrc} />
-                <span>Spenden</span>
-              </a>
-            </div>
-          </div>
-        </div>
-      </div>
+              </Col>
+              <Col xs> 
+                <a href="#" className="icon">
+                  <img alt="part" src={donateSrc} />
+                  <span>Spenden</span>
+                </a>
+              </Col>
+            </Row>
+          </Col>
+        </BrandingWrap>
+      </BrandingGrid>
     )
   }
 }
+
+
+const BrandingGrid = styled(Grid)`
+  padding: 0;
+`
+
+const BrandingWrap = styled(Row)`
+
+  margin-right: 0;
+
+  a:not(.icon), a.icon {
```

Comment:
In my opinion: way more components, I'd use the selectors pretty much only if I can't avoid it (e.g. because I have third-party stuff that works with classnames or where I don't have direct access to). Otherwise, Components all the way since they are better composable, type-safe, reusable and I can jump from the component to its styles.